DataProcessing/process.sample.py

```diff
@@ -0,0 +1,2 @@
+# CLRImports is required to handle Lean C# objects for Mapped Datasets (Single asset and Universe Selection)
+from CLRImports import *
```

Comment:
Would be nice to add an example of how to map a ticker to a symbol for a given date?